src/modules/lorem/index.ts

```diff
@@ -332,7 +359,46 @@ export class LoremModule {
 
     const method = this.faker.helpers.arrayElement(methods);
 
-    return `${this[method]()}`;
+    if (typeof options === 'number') {
+      options = { length: options };
+    }
+
+    if (options.length == null) {
+      return `${this[method]()}`;
+    }
+
+    if (typeof options.length === 'number' && options.length < 0) {
+      throw new FakerError(
+        `Length ${options.length} should be a non-negative integer.`
+      );
+    }
+
+    if (typeof options.length !== 'number') {
+      if (options.length.min != null && options.length.min < 0) {
+        throw new FakerError(
+          `Min ${options.length.min} should be a non-negative integer.`
+        );
+      }
+
+      if (options.length.max != null && options.length.max < 0) {
+        throw new FakerError(
+          `Max ${options.length.max} should be a non-negative integer.`
+        );
+      }
+    }
+
+    const length = this.faker.helpers.rangeToNumber(options.length);
+
+    if (length === 0) {
+      return '';
+    }
+
+    let text = `${this[method]()}`;
+    while (text.length <= length) {
+      text = `${text} ${this[method]()}`;
+    }
```

Comment:
It is also possible to write this as a do-while-loop, but I'm not sure about that:

```suggestion
    let text = '';
    do {
      text = `${text}${this[method]()} `;
    } while (text.length <= length);
```

Reply:
What is the advantage of doing is this way?

Reply:
There are two tiny benefits:

1. The trailing ` ` adds one to the text.length and thus terminates the loop potentially early (this would also work in the normal while loop)
2. The method is invoked in only one location

With the tiny drawback that there is one additional operation with the empty string